test/blackbox-tests/test-cases/unreadable-target.t

```diff
@@ -14,6 +14,6 @@
   2 |   (targets a b)
   3 |   (action (bash "echo content > a; chmod -r a; ln -s foo b")))
   Error: Error trying to read targets after a rule was run:
-  - a: Permission denied
+  - a: open(_build/default/a): Permission denied
```

Comment:
Looks like a regression, can we fix this?

Reply:
Sure, it's fixed. Although if this a regression, we have many other error messages like it.

Reply:
Thanks! I surely prefer turning bad error messages into good ones rather than the other way around :)